fetch.bs

```diff
@@ -3122,10 +3122,17 @@ steps:
 
 <li>
   <p>If <var>contentLengthValue</var> is non-null, <var>httpRequest</var>'s
-  <a>keepalive flag</a> is set, and <var>contentLengthValue</var> is greater than a
-  user-agent-defined maximum, then return a <a>network error</a>.
+  <a>keepalive flag</a> is set:
 
-  <p class="note no-backref">The above user-agent-defined maximum ensures that requests that are
+  <ul>
+    <li>Let <var>group</var> be <var>httpRequest</var>'s <a>client</a>'s <a>fetch group</a>.
+    <li>Let <var>inflight keepalive bytes</var> be the sum of <a>request</a> <a>body</a>'s
+    <a>total bytes</a> for each <a>record</a> in <var>group</var> whose <a>done flag</a> is unset.
```

Comment:
I get what you're trying to say, but this sentence does not read well. Maybe "sum of each X's Y in Z"?

I also think this is lacking some `for=""` attributes since the request you're talking about here is that of the fetch group, as is the record.